### ISSUE TITLE
Looking for a co-maintainer

### ISSUE THREAD

**Issue by tcoopman** [tcoopman/image-webpack-loader#47]
My time is very limited at the moment and I can't give this package the love it needs.
If someone is interested in helping maintain this package, please give a shout!

I don't have any expectations at the moment, so just let me know what you want to do. This package doesn't require lot's of maintenance, but someone who can keep an eye to updated dependencies and bugs would help.

I already gave @pavelloz acces, but if anyone else wants to help out that would be great!

**Comment by anikethsaha** [comment]
@tcoopman I would love to help on this if it's still open 👍

**Comment by tcoopman** [comment]
It is still open but not urgent at the moment. The maintenance at the moment is rather low, so this means that there is not much work here.
I've added dependabot, so dependencies are taken care of. For other issues or feature requests I ask that people provide a PR. So if there are any issues requiring fixes, you're welcome to go over them (unfortunately most of the issues are about dependencies not installing, so not something we can easily fix).
I'm not open for changing a lot though, the api is simple and quite stable and I want to keep it that way.

So if you're interested in helping maintaining issues and PRs, please go ahead. If it's useful we can look at giving you access later as well.

**Comment by anikethsaha** [comment]
Yeah agree 👍 its more like a module package so I guess it stable pretty much.

> So if you're interested in helping maintaining issues and PRs, please go ahead. If it's useful we can look at giving you access later as well.

Sure.
Also if you need help with reviewing, you can ping me too. I am looking for a few spaces here to check if anything is required to change or not 😃
